### ISSUE TITLE
Enable sync without manually unblocking

### ISSUE THREAD

**Issue by mattgmarcus** [LedgerSync/ledger_sync#11]
You currently have to manually unblock a sync in order for it to happen. We'd like the ability to specify (whether through config or the API call) that the sync will just happen

**Comment by ryanwjackson** [comment]
@mattgmarcus 

To clarify, we currently require manual intervention if a resource is to be created and not just updated.  Right now that has to happen by the user through the UI.  I believe our options are the following:

1. Add app config to always create without confirmation
2. Add per-account option to create without confirmation
3. Add per-sync option to create without confirmation

(1) and (2) (though not as much) seem kind of dangerous, as I think it's better to make the provider be explicit with that action as it could result in duplicates.  Is (3) enough to solve the problem?

If so, we can add a `skip_resource_creation_confirmations` (welcome better name) to the syncs endpoint.

**Comment by mattgmarcus** [comment]
Yeah I'm on board for #3. We can always bubble up the setting later if we want to. 
That name seems fine for now
